tut2_EOFM_Evaluation.ipynb

```diff
@@ -107,9 +136,28 @@
     "from terratorch import BACKBONE_REGISTRY"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "79667161-8062-4ca3-af36-2f64af6905fe",
+   "metadata": {},
+   "source": [
+    "To make a `.netrc` file, run the following, replacing your Earth Data username and password accordingly:"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
+   "id": "6420fb76-5eac-4386-a5e7-0b3b0605d30c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!echo \"machine urs.earthdata.nasa.gov login your_username password your_password\" > ~/.netrc\n",
+    "!chmod 0600 ~/.netrc"
```

Comment:
The [earthaccess.login(persist=True)](https://earthaccess.readthedocs.io/en/latest/user_guide/authenticate/#login-using-a-netrc) line will create a netrc file, so no need to explicitly add this.